docs/AutomationReference.md

```diff
@@ -123,26 +123,37 @@ The `ScannerFactory.CreateScanner` method returns an `IScanner` object.
 
 #### IScanner
 
+##### `ScanAsync`
+The `ScanAsync` method asynchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner. This method should be `await`ed.
+
 ##### `Scan`
-The Scan runs AxeWindows automated tests using the config provided at the time of creation of the scanner.
+The `Scan` method synchronously runs AxeWindows automated tests using the config provided at the time of creation of the scanner, and blocks until the scan is complete.
 
-###### Parameters
+##### Parameters
```

Comment:
Docs usually have an API, then information about the API as children. Links as children are normal and expected. As currently drafted, ScanAsync has no information and it's all in siblings. Can we put Parameters and Return object as H6 children of each of the H5 Scan and the H5 ScanAsync, then they can link to the types? it might make sense to have an H5 Objects and list the ScanOutput and ScanOptions--along with the other types--there. I